.gitignore

```diff
@@ -160,4 +160,8 @@ crates/rolldown_binding/artifacts/
 packages/**/THIRD-PARTY-LICENSE
 
 # Opinionated configuration
-.vscode/launch.json
\ No newline at end of file
+.vscode/launch.json
```

Comment:
Could you delete this one?

Reply:
![image](https://github.com/rolldown/rolldown/assets/17974631/3628f72a-1c6c-457e-ae4b-3267a98bce39), 
Ok,but it is not added by me.